README.md

```diff
@@ -9,7 +9,7 @@ Current Stage:
 * Gabriel Isenberg (@the_gisenberg)
 
 ## Overview and motivation
-When looking for a property value deeply in a tree structure, one has often to check whether intermediate nodes exist:
+When looking for a property value deep in a tree-like structure, one often has to check whether intermediate nodes exist:
```

Comment:
“deeply” is correct here; the other changes look good

Reply:
That depends -- are you "looking deeply" for a property value, or are you looking for a property value that is "deep" in the structure?

Reply:
Fair point; i guess I’d thought of the former but the latter is also true.

Reply:
Yeah, I'd agree "looking deeply" is possible, but "looking for a value that is deep in the structure" sounds more fitting in this situation, I think.

Would it help if instead it read, "When looking for a property value **that's** deep in a tree-like structure, one often has to check whether intermediate nodes exist:"?

Reply:
Sounds great!